### PR TITLE
[zh-cn]: update the translation of HTML `<header>` element

### PR DESCRIPTION
**Pull request by T34-active**
### Description
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header
### Related issues and pull requests
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/header/index.md
* Last commit: https://github.com/mdn/content/commit/6336af7a3880c350919e5b29f83b938fb1594362

### CODE REVIEW

File: files/zh-cn/web/html/element/header/index.md [hunk 1]
```diff
@@ -1,79 +1,160 @@
...
-  </tr>
-  <tr>
-   <th scope="row">DOM 接口</th>
-   <td>{{domxref("HTMLElement")}}</td>
-  </tr>
- </tbody>
-</table>
+## 使用说明
+
+`<header>` 元素的意义与网站范围内的 [`banner`](/zh-CN/docs/Web/Accessibility/ARIA/Roles/banner_role) 地标角色相同，除非嵌套在分段内容内。然而，`<header>` 元素就不是一个地标。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/header/index.md:16]: 最后一句话的翻译我认为是错误的。因为英文文档用的是“Then”，而不是“However”或者“While”。请仔细斟酌。

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/html/element/header/index.md (hunk 1 above)
Comment (T34-active) [inline files/zh-cn/web/html/element/header/index.md:16]:
```suggestion
`<header>` 元素的意义与网站范围内的 [`banner`](/zh-CN/docs/Web/Accessibility/ARIA/Roles/banner_role) 地标角色相同，除非嵌套在分段内容内。在这种情况下，`<header>` 元素不再是地标。
```

File: files/zh-cn/web/html/element/header/index.md [hunk 2]
```diff
@@ -1,79 +1,160 @@
...
+        >
+        或
+        <code
+          ><a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/region_role"
+            >region</a
+          ></code
+        >元素。则为 <a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/banner_role">banner</a
+        > 或 <a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/generic_role"
+          >generic</a
+        >。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/header/index.md:142]: 翻译对吗（考虑一下前面的内容）

**Review by yin1999 — APPROVED** [review]
感谢 🎉

**Pull request merged**